src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx

```diff
@@ -284,6 +287,31 @@ export default function MeasureInformation() {
             />
           </div>
 
+          <br />
+          <DisplayDiv>
+            <span className={classes.asterisk}>*</span>
+            <span tw="mr-2" className={classes.fieldName}>
+              ECQM Abbreviated Title:
+            </span>
+          </DisplayDiv>
+          <div className={formRowGapped}>
+            <TextField
+              placeholder="Enter ECQM Abbreviated Title"
```

Comment:
this placeholder is different from create measure modal ecqm placeholder

Reply:
Thanks will change it.

Reply:
This is done.